server/src/http/unleash_client.rs

```diff
@@ -756,7 +756,7 @@ mod tests {
                     ),
                 |_| AppConfig::default(),
             ))
-            .rustls_021_with_config(server_config, tls_acceptor_config)
+            .rustls_0_22_with_config(server_config, tls_acceptor_config)
```

Comment:
This isn't really useful to this PR but I'm just fascinated that a function has a version string in the name